guardrails/cli/hub/install.py

```diff
@@ -242,22 +242,17 @@ def do_nothing_context(*args, **kwargs):
     with loader(dl_deps_msg, spinner="bouncingBar"):
         install_hub_module(module_manifest, site_packages, quiet=quiet)
 
-    if local_models is True or local_models is False:
-        install_local_models = local_models
-    else:
-        try:
-            if module_manifest.tags and module_manifest.tags.has_guardrails_endpoint:
-                install_local_models = typer.confirm(
-                    "This validator has a Guardrails AI inference endpoint available. "
-                    "Would you still like to install the"
-                    " local models for local inference?",
-                )
-            else:
-                install_local_models = typer.confirm(
-                    "Would you like to install the local models?", default=True
-                )
-        except AttributeError:
-            install_local_models = False
+    install_local_models = local_models
+
+    try:
+        if module_manifest.tags and module_manifest.tags.has_guardrails_endpoint:
```

Comment:
can we also add to this check that the guardrails config file does not contain `use_remote_inferencing=true`? If `use_remote_inferencing=true`, we should not show this and just assume that they want to use remote inferencing - i.e. not run post-install. 

If they really wanna run post-install/use local models, they can pass a flaaaaag